UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.UnitTests/Services/ProductDataServiceTests.cs

```diff
@@ -0,0 +1,78 @@
+﻿using FakeItEasy;
+using FluentValidation.Results;
+using NUnit.Framework;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using UKHO.ExchangeSetService.API.Services;
+using UKHO.ExchangeSetService.API.Validation;
+using UKHO.ExchangeSetService.Common.Models.Request;
+using UKHO.ExchangeSetService.Common.Models.Response;
+
+namespace UKHO.ExchangeSetService.API.UnitTests.Services
```

Comment:
Again I am not convinced these are unit tests.

Reply:
Did changes accordingly as added request model to respective unit test for WhenValidProductVersionRequest_ThenCreateProductDataByProductVersionsReturnsOkrequest